src/views/Change.vue

```diff
@@ -203,13 +203,14 @@ export default class Change extends Mixins(CommonMixin, FeeMixin, FilingTemplate
       // fetch entity snapshot
       const entitySnapshot = await this.fetchEntitySnapshot()
 
-      switch (entitySnapshot.businessInfo.legalType) {
-        case CorpTypeCd.BENEFIT_COMPANY:
-        case CorpTypeCd.BC_COMPANY:
-        case CorpTypeCd.BC_CCC:
-        case CorpTypeCd.BC_ULC_COMPANY:
-          throw new Error(`Invalid entity type, must not be benefit company id`)
+      switch (entitySnapshot?.businessInfo?.legalType) {
+        case CorpTypeCd.SOLE_PROP:
+        case CorpTypeCd.PARTNERSHIP:
+          break // acceptable types
+        default:
+          throw new Error(`Invalid entity type, must be a firm (Sole Prop or General Partnership)`)
```

Comment:
Can you show a screenshot of this message being displayed please?

Reply:
<img width="1371" alt="image" src="https://github.com/bcgov/business-edit-ui/assets/52959626/638f3ff7-9c3e-47aa-935e-6a608326522f">

Reply:
LGTM